content/200-concepts/100-components/01-prisma-schema/index.mdx

```diff
@@ -138,7 +138,12 @@ Run prisma generate to generate Prisma Client.
 
 ## Accessing environment variables from the schema
 
-Environment variables help you to **keep secrets out of the schema file** while **improving its portability**. They can be accessed using the `env` function:
+Environment variables can be used to provide configuration values in the schema, such as database connection urls.
+
+By using environment variables instead of hardcoding values, you can **improve the portability of the schema** file, enabling you to use it in different environments.
+This has the added benifit of **keeping secrets out of the schema file**, as secrets should never be hardcoded, shared or committed.
+
+They can be accessed using the `env` function:
```

Comment:
Do we describe functions with or without `()` usually? A few lines below we use `env()`.